### PR TITLE
feat: Add ConformancePackRuleCompliances

### LINKED ISSUE #616: Ensure Support for AWS::Config::*
- [ ] AWS::Config::ConformancePackCompliance
- [x] ~~AWS::Config::ResourceCompliance~~

### PR DESCRIPTION
**Pull request by spangenberg**
Fixes #616

### CODE REVIEW

File: resources/provider/provider.go [hunk 1]
```diff
@@ -82,157 +82,158 @@ func Provider() *provider.Provider {
...
-			"wafv2.web_acls":                        wafv2.Wafv2WebAcls(),
-			"wafregional.rate_based_rules":          wafregional.RateBasedRules(),
-			"wafregional.rule_groups":               wafregional.RuleGroups(),
-			"wafregional.rules":                     wafregional.Rules(),
-			"wafregional.web_acls":                  wafregional.WebAcls(),
-			"workspaces.workspaces":                 workspaces.Workspaces(),
-			"workspaces.directories":                workspaces.Directories(),
-			"xray.encryption_config":                xray.EncryptionConfigs(),
-			//"iot.security_profiles": 				 iot.IotSecurityProfiles(), //TODO disabled because of api error NotFoundException: No method found matching route security-profiles for http method GET.
+			"accessanalyzer.analyzers":                 accessanalyzer.AccessAnalyzerAnalyzers(),
```
Comment (disq) [inline resources/provider/provider.go:85]: Maybe we should add newline after some resource "class"es? wdyt?

File: resources/services/config/conformance_pack_rule_compliance.go [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+
+	"github.com/aws/smithy-go"
+)
+
+func ConformancePackRuleCompliance() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_config_conformance_pack_rule_compliances",
```
Comment (disq) [inline resources/services/config/conformance_pack_rule_compliance.go:19]: Why not add this as a subresource to `config.conformance_packs`? this way you'd get rid of the 4-level resolver

File: resources/provider/provider.go (hunk 1 above)
Comment (spangenberg) [reply resources/provider/provider.go:85]: Good point, I'll do it at least for the `config`.

File: resources/services/config/conformance_pack_rule_compliance.go (hunk 2 above)
Comment (spangenberg) [reply resources/services/config/conformance_pack_rule_compliance.go]: done

**Review by disq — APPROVED** [review]
LGTM

**Pull request merged**